include/TrustWalletCore/TWCoinType.h

```diff
@@ -29,15 +29,15 @@ enum TWCoinType {
     TWCoinTypeBitcoinCash = 145,
     TWCoinTypeBitcoinGold = 156,
     TWCoinTypeCallisto = 820,
-    TWCoinTypeCardano = 1815, // Note: Cardano Shelley testnet uses purpose 1852 (not 44) 1852/1815
+    TWCoinTypeCardano = 1815,
```

Comment:
```suggestion
    TWCoinTypeCardano = 1815, // Note: Cardano Shelley testnet uses purpose 1852 (not 44) 1852/1815
```

Reply:
This comment looks confusing to me, we won't support its testnet anyway; if you really need it, maybe put it somewhere else?